assets/js/blocks/checkout/inner-blocks/checkout-payment-block/edit.tsx

```diff
@@ -85,6 +86,7 @@ export const Edit = ( {
 						</ExternalLink>
 					</PanelBody>
 				) }
+				<CheckoutSidebarCompatibilityNotice />
```

Comment:
Same here! We should place the alert under the Name & description panel before the Block's settings panels (i.e., Let's move it to line 56).